app/router.go

```diff
@@ -133,8 +133,15 @@ func RegisterReportPostHandler(a Adder, router *mux.Router) {
 
 		decoder := gob.NewDecoder(reader).Decode
```

Comment:
Backwards compatibility with 0.12. I will create a techdebt ticket to remove it later on.